api/organisations/tasks.py

```diff
@@ -290,6 +290,17 @@ def restrict_use_due_to_api_limit_grace_period_over() -> None:
         if not stop_serving and not block_access:
             continue
 
+        if not organisation.has_subscription_information_cache():
+            continue
+
+        subscription_cache = organisation.subscription_information_cache
+        api_usage = get_current_api_usage(organisation.id, "30d")
+        if api_usage / subscription_cache.allowed_30d_api_calls < 1.0:
+            logger.info(
+                f"API use for organisation {organisation.id} has fallen to below limit, so not restricting use."
+            )
```

Comment:
Do we _need_ to log that? I'd rather log the other branch?

Reply:
This is the less frequently occurring case where we may need to debug when the task didn't select them, so I think it's a good candidate for inclusion.